doc/developers/advanced_installation.rst

```diff
@@ -305,22 +302,9 @@ Ubuntu:
 
     sudo apt-get install build-essential python3-dev python3-pip
 
-then proceed as usual:
 
-.. prompt:: bash $
-
-    pip3 install cython
-    pip3 install --editable . \
-        --verbose --no-build-isolation \
-        --config-settings editable-verbose=true
-
-Cython and the pre-compiled wheels for the runtime dependencies (numpy, scipy
-and joblib) should automatically be installed in
-``$HOME/.local/lib/pythonX.Y/site-packages``. Alternatively you can run the
-above commands from a virtualenv_ or a `conda environment`_ to get full
-isolation from the Python packages installed via the system packager. When
-using an isolated environment, ``pip3`` should be replaced by ``pip`` in the
-above commands.
+If, for some reason, you are not using an isolated environment (which is not recommended), ``pip3`` needs to be used instead of ``pip`` in the
+following command above.
```

Comment:
"following [...] above" is contradictory. And there's actually no command above or below 😄.

```suggestion
build command.
```
Maybe add a reference to the section where we detail the build command

Reply:
> Maybe add a reference to the section where we detail the build command

We could even reference it at the end of each of these sections.